services/src/modules/arguments-injection/arguments-injection.ts

```diff
@@ -25,14 +27,15 @@ function injectTemplate<T = unknown>(
     exports: info && context && getExportsProxy(context.exports, info?.parentType, source as Record<string, unknown>),
     vars: info?.variableValues,
     info,
+    pluginData: context?.resourceGroup?.pluginData,
```

Comment:
`pluginData` is a bit long.
```suggestion
    plugins: context?.resourceGroup?.pluginData,
```